src/pages/reviewList.jsx

```diff
@@ -1,30 +1,34 @@
-import { useEffect, useState } from "react";
+import { useEffect, useState} from "react";
+import { useParams } from "react-router-dom";
 import Loading from "../components/Loading";
 import ReviewCard from "../components/ReviewCard";
 import axios from "axios";
 import CategoryBar from "../components/CategoryBar";
 
 const ReviewList = () => {
-  const [reviews, setReviews] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
+  const [reviews, setReviews] = useState([]);
+  const {slug} = useParams();
 
   useEffect(() => {
     setIsLoading(true);
-
-    
-
+    console.log(slug)
+    let reviewURL = `https://be-games-project-tm.herokuapp.com/api/reviews`
+    if(slug) {reviewURL = reviewURL + `?category=${slug}`}
+    console.log(reviewURL)
```

Comment:
I'd look at putting all your API requests in a single file - id also try use the the axios params object to handle queries - will make it much easier later on

Reply:
Have refactored to use axios params, will extract the api request into a separate file once there is more than one, since I have several more tickets to complete before I will reach the point of adding another